bin/midi.ml

```diff
@@ -50,6 +50,13 @@ let bend_pitch ~bend ~timestamp =
   let data2 = char_of_int (bend lsr 7) in
   Event.create ~status:'\224' ~data1 ~data2 ~timestamp
 
+let control_change ~cc ~value ~timestamp =
+  if cc > 119 then raise (invalid_arg "Sorry, [cc] must be <= 119")
```

Comment:
You can just do `invalid_arg ` here, no need to use `raise`. (`invalid_arg` does it by itself)

Reply:
Ok